rs/consensus/src/idkg/signer.rs

```diff
@@ -551,11 +553,17 @@ impl ThresholdSigner for ThresholdSignerImpl {
             .get_state()
             .signature_request_contexts()
             .iter()
-            .flat_map(|(callback_id, context)| {
-                context.matched_pre_signature.map(|(_, height)| RequestId {
+            .flat_map(|(callback_id, context)| match &context.args {
+                ThresholdArguments::Ecdsa(_) | ThresholdArguments::Schnorr(_) => {
```

Comment:
Maybe also implement `is_idkg()` and `is_vetkd()` functions for `TresholdArguments`? Makes for cleaner code